openapi.yaml

```
@@ -8960,6 +8960,15 @@ paths:
                     will not be performed.
                   example: false
                   default: false
+                migration_type:
```

```suggestion
                type:
```
This is just "type" at this endpoint, the other endpoint is correct with "migration_type". Can we add a note here or above that warm is not an available type for cross DC migrations? It's currently only available for pending migrations.